cli/graph_util.rs

```diff
@@ -719,6 +729,36 @@ impl deno_graph::source::Reporter for FileWatcherReporter {
   }
 }
 
+pub fn workspace_config_to_workspace_members(
+  workspace_config: &deno_config::WorkspaceConfig,
+) -> Result<Vec<deno_graph::WorkspaceMember>, AnyError> {
+  workspace_config
+    .members
+    .iter()
+    .map(workspace_member_config_try_into_workspace_member)
+    .collect()
+}
+
+fn workspace_member_config_try_into_workspace_member(
+  config: &deno_config::WorkspaceMemberConfig,
+) -> Result<deno_graph::WorkspaceMember, AnyError> {
+  let nv = deno_semver::package::PackageNv {
+    name: config.package_name.clone(),
+    version: deno_semver::Version::parse_standard(&config.package_version)?,
```

Comment:
Perhaps we should say that it was the config file's version that failed to parse when this fails? (ex. via `context`/`with_context`)